addons/im_livechat/static/src/core/channel_member_list_patch.xml

```diff
@@ -5,9 +5,9 @@
             <div class="d-flex flex-column">
                 <t>$0</t>
                 <div class="ms-2 d-flex">
-                    <span t-if="member.persona?.lang_name" class="me-2">
+                    <span t-if="member.getLangName()" class="me-2">
                         <i class="fa fa-comment-o me-1" aria-label="Lang"/>
-                        <t t-esc="member.persona.lang_name"/>
+                        <t t-esc="member.getLangName()"/>
                     </span>
```

Comment:
Preferably it should show lang only for livechat visitors.

I see that `lang_name` on persona currently is set only in livechat visitor flow, but if we change code to receive it somehow, I'm pretty sure we don't want the side-effect to see the language of all internal users in the list. I prefer that if this becomes an explicitly choice, then we should actively implement it at that time, rather than having an accidental functional change

Reply:
updated :+1: